src/earn/EarnCollectScreen.tsx

```diff
@@ -0,0 +1,250 @@
+        size={BtnSizes.FULL}
+        text={t('earnFlow.collect.cta')}
+        onPress={onPress}
+        testID="EarnCollectScreen/CTA"
+      />
+    </SafeAreaView>
+  )
+}
+
+function CollectItem({
+  tokenInfo,
+  rewardAmount,
+  title,
+}: {
+  tokenInfo: TokenBalance
+  rewardAmount: BigNumber
+  title: string
+}) {
+  return (
+    <>
+      <Text style={styles.collectItemTitle}>{title}</Text>
+      <View style={styles.row}>
+        <View style={styles.iconContainer}>
+          <TokenIcon token={tokenInfo} size={IconSize.LARGE} />
+        </View>
+        <View>
+          <TokenDisplay
+            style={styles.cryptoText}
+            tokenId={tokenInfo.tokenId}
+            amount={rewardAmount || tokenInfo.balance}
+            showLocalAmount={false}
+          />
+          <TokenDisplay
+            style={styles.fiatText}
+            tokenId={tokenInfo.tokenId}
+            amount={rewardAmount || tokenInfo.balance}
+            showLocalAmount={true}
+          />
+        </View>
+      </View>
+    </>
+  )
+}
+
+function Rate({ tokenInfo, poolApy }: { tokenInfo: TokenBalance; poolApy: number }) {
+  const { t } = useTranslation()
+  return (
+    <View>
+      <Text style={styles.rateText}>{t('earnFlow.collect.rate')}</Text>
+      <View style={styles.row}>
+        <TokenIcon token={tokenInfo} size={IconSize.SMALL} />
+        <View style={styles.apyContainer}>
+          <Text style={styles.apyText}>
+            {t('earnFlow.activePools.apy', {
+              apy: poolApy,
+            })}
+          </Text>
+        </View>
+      </View>
+    </View>
+  )
+}
+
+function GasFee({
+  maxFeeAmount,
+  feeCurrency,
+}: {
+  maxFeeAmount: BigNumber
+  feeCurrency: TokenBalance
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <View>
+      <Text style={styles.rateText}>{t('earnFlow.collect.fee')}</Text>
+      <TokenDisplay
+        style={styles.apyText}
+        tokenId={feeCurrency.tokenId}
+        amount={maxFeeAmount}
+        showLocalAmount={false}
+      />
+      <TokenDisplay
+        style={styles.gasFeeFiat}
+        tokenId={feeCurrency.tokenId}
+        amount={maxFeeAmount}
+        showLocalAmount={true}
+      />
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  contentContainer: {
+    padding: Spacing.Thick24,
+  },
+  container: {
+    flex: 1,
+    justifyContent: 'space-between',
+  },
+  title: {
+    ...typeScale.titleMedium,
+    marginBottom: Spacing.Thick24,
+  },
+  collectInfoContainer: {
+    padding: Spacing.Regular16,
+    borderColor: Colors.gray2,
+    borderWidth: 1,
+    borderRadius: 16,
+    backgroundColor: Colors.gray1,
+  },
+  row: {
+    flexDirection: 'row',
+    gap: Spacing.Smallest8,
+  },
+  cryptoText: {
+    ...typeScale.labelSemiBoldLarge,
+    color: Colors.black,
+  },
+  fiatText: {
+    ...typeScale.bodySmall,
+    color: Colors.gray4,
+  },
+  collectItemTitle: {
+    ...typeScale.labelSemiBoldXSmall,
+    marginBottom: 4,
```

Comment:
use Spacing.Tiny4 here and below. Also, some of the padding looks off from the figma